### ISSUE TITLE
Attribute::DefaultForegroundColor has wrong constant

### ISSUE THREAD

**Issue by thomcc** [TimonPost/crossterm#135]
According to https://en.wikipedia.org/wiki/ANSI_escape_code#SGR_parameters this should be 39. I can confirm that as-is it does not work on iTerm2 on macOS. I'd PR but I have no access to test on anything other than my machine at the moment.

https://github.com/TimonPost/crossterm/blob/0bfebb338de11ccd4faed8720d0cff06399af794/crossterm_style/src/enums/attribute.rs#L118

(I also feel like this belongs as a Color, even though the escape code is rather different, since concepturally it's for turning off colors. That's entirely separate though).

**Comment by TimonPost** [comment]
Yes, you are right, great catch!

**Comment by TimonPost** [comment]
You are right that this should be done with `Color` fun enought, today, I made the follwing changes:

```
    println!("{}", Colored::Fg(Color::Reset));
    println!("{}", Colored::Bg(Color::Reset));
```
I want to make the resetting of colors with the above API possible, it is already working.

**Comment by TimonPost** [comment]
#138

**Comment by TimonPost** [comment]
Fixed in #138
